pkg/sidecars/server/server.go

```diff
@@ -0,0 +1,117 @@
+package sidecarserv
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/inspr/inspr/pkg/sidecars/models"
+	"go.uber.org/zap"
+)
+
+// Server is a struct that contains the variables necessary
+// to handle the necessary routes of the rest API
+type Server struct {
+	broker       string
+	Reader       models.Reader
+	Writer       models.Writer
+	inAddr       string
+	outAddr      string
+	client       *http.Client
+	runningRead  bool
+	runningWrite bool
+}
+
+var logger *zap.Logger
+
+func init() {
+	logger = zap.NewNop()
```

Comment:
Should probably return this to `.NewProduction`. Also remember to change from "lbsidecar" to "kafka-sidecar" or something